cli/src/main/java/org/jboss/sbomer/cli/feature/sbom/processor/DefaultProcessor.java

```diff
@@ -277,12 +277,12 @@ public Bom process(Bom bom) {
         // If there are any purl relcoations, process these.
         purlRelocations.forEach((oldPurl, newPurl) -> updatePurl(bom, oldPurl, newPurl));
 
-
         if (bom.getMetadata() != null && bom.getMetadata().getComponent() != null) {
             Component mainComponent = bom.getMetadata().getComponent();
             addMissingNpmDependencies(bom, mainComponent);
             // Add missing NPM Depenencies for CycloneDxGenerateOperationComand manifest
-            if (mainComponent.getDescription() != null && mainComponent.getDescription().contains(SBOM_REPRESENTING_THE_DELIVERABLE)) {
+            if (mainComponent.getDescription() != null
+                    && mainComponent.getDescription().contains(SBOM_REPRESENTING_THE_DELIVERABLE)) {
```

Comment:
Is this just formating? Shoudl I be doing something on my side to produce properly formated code?

Reply:
It is, but I don't recall making any particular setting.... I see we have the `formatter-maven-plugin` configured, but that is it...